src/main/resources/assets/carpet/scripts/shapes.scl

```diff
@@ -1,15 +1,20 @@
 //This is a library app, mostly for the draw command, which allows to import these shape-drawing functions into your own app.
-//Arguments for each are to be given as a singleton list, for ease of use. This is with the exception of the fill_flat functions, with
-//the assumption that they're unlikely to be used outside of internal calls, but it is still available to be called if necessary.
-//Make sure you read the arguments and input them in the correct order, or it will mess up.
-//these functions return a set of all the positions in the shape, given the parameters, without repeating a position.
+//The _fill_flat functions are mostly used in internal calls, but can be used outside as well
```

Comment:
Maybe don't suggest that (using them). That way it's implied it's not something supported and it can be changed or removed without breaking anything.

Reply:
k